.github/workflows/license_check.yml

```diff
@@ -10,6 +10,8 @@ jobs:
   check-headers:
     name: Check that license headers are in place
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
```

Comment:
📣 This is enough to cause the `LICENSE` checks to fail when needed - https://github.com/slackapi/slack-cli/pull/54/commits/6591d1f7bcad9cd982941deee6d9a1e9173e2b81 causes:

https://github.com/slackapi/slack-cli/actions/runs/14578766377/job/40890636282?pr=54